pkg/asset/manifests/clusterapi/cluster.go

```diff
@@ -156,7 +158,13 @@ func (c *Cluster) Generate(dependencies asset.Parents) error {
 		var err error
 		out, err = azure.GenerateClusterAssets(installConfig, clusterID)
 		if err != nil {
-			return errors.Wrap(err, "failed to generate AWS manifests")
+			return errors.Wrap(err, "failed to generate Azure manifests")
+		}
+	case gcptypes.Name:
+		var err error
+		out, err = gcp.GenerateClusterAssets(installConfig, clusterID)
+		if err != nil {
+			return errors.Wrap(err, "failed to generate GCP manifests")
```

Comment:
Let's not use `errors.Wrap` in new code:
```suggestion
			return fmt.Errorf("failed to generate GCP manifests: %w", err)
```